client/src/components/HomePage.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from "react"
+
+const HomePage = (props) => {
+  const [ownedIngredients, setOwnedIngredients] = useState([])
+  const [recipes, setRecipes] = useState([])
+
+  const fetchIngredients = async () => {
+    if(props.user) {
+      try {
+        const response = await fetch(`/api/v1/ownedIngredients/${props.user.id}`)
+        const body = await response.json()
+        setOwnedIngredients(body.ownedIngredients)
+      } catch(err) {
+        console.error(`Error in fetch: ${err.message}`)
+      }
+    }
+  }
+
+  const fetchRecipes = async () => {
+    if(props.user) {
+      try {
+        const response = await fetch(`/api/v1/recipes/${props.user.id}`)
+        const body = await response.json()
+        setRecipes(body.recipes)
+      } catch(err) {
+        console.error(`Error in fetch: ${err.message}`)
+      }
+    }
+  }
```

Comment:
you could make one less fetch request by making a custom endpoint that returns both the recipes and ingredients for a user!